devices/Mpu6886/Mpu6886.cs

```diff
@@ -0,0 +1,176 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers.Binary;
+using System.Device.I2c;
+using System.Device.Model;
+using System.IO;
+using System.Numerics;
+using System.Threading;
+
+namespace Iot.Device.Mpu6886
+{
+    /// <summary>
+    /// Mpu6886 accelerometer and gyroscope
+    /// </summary>
+    [Interface("Mpu6886 accelerometer and gyroscope")]
+    public class Mpu6886AccelerometerGyroscope : IDisposable
+    {
+        private I2cDevice _i2c;
+        private const double gyroscopeResolution = 2000.0 / 32768.0;
+        private const double accelerometerResolution = 8.0 / 32768.0;
+        /// <summary>
+        /// Mpu6886 - Accelerometer and Gyroscope bus
+        /// </summary>
+        public Mpu6886AccelerometerGyroscope(
+            I2cDevice i2cDevice)
+        {
+            _i2c = i2cDevice ?? throw new ArgumentNullException(nameof(i2cDevice));
+
+            SpanByte readBuffer = new byte[1];
+
+            _i2c.WriteByte((byte)Mpu6886.Register.WhoAmI);
+            _i2c.Read(readBuffer);
+            if (readBuffer[0] != 0x19)
+                throw new IOException($"This device does not contain the correct signature 0x19 for a MPU6886");
+
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.PowerManagement1, 0x00 }));
+            Thread.Sleep(10);
+
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.PowerManagement1, 0x01 << 7 }));
```

Comment:
rather than 0x01 << 7 just directly use the right 0b01000_0000. It's much easier to read and understand or directly 0x80. 
In most cases doing the << doesn't bring any value and do not bring clarity neither for constants.